corehq/apps/app_manager/templates/app_manager/module_view.html

```diff
@@ -250,9 +250,9 @@
                         <input type="text"
                             placeholder="{% trans "XPath expression" %}"
                             name="module_filter"
-                            value="{{ module.module_filter }}"/>
+                            value="{% if module.module_filter %}{{ module.module_filter }}{% endif %}"/>
```

Comment:
why not use a `|default=""` filter up here too? Looks like it would have the same outcome according to the [docs](https://docs.djangoproject.com/en/1.7/ref/templates/builtins/#default).